### PR TITLE
UI: new combobox, jointly-or-in-concert-control table, and style updates

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/20757
*Issue:*https://github.com/bcgov/entity/issues/20756

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-167-gsp7s372.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-167-gsp7s372.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-167-gsp7s372.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-167-gsp7s372.web.app

File: btr-web/btr-main-app/interfaces/btr-bods/btr-bods-person-i.ts [hunk 1]
```diff
@@ -37,4 +37,11 @@ export interface BtrBodsPersonI {
   missingInfoReason?: string
 
   uuid: string
+
+  sharesInConcert?: string[],
```
Comment (kialj876) [inline btr-web/btr-main-app/interfaces/btr-bods/btr-bods-person-i.ts:41]: maybe we can meet on this, but doesn't it make sense for this data to be inside the BodsInterestI ? Having it here makes it seem unrelated to the control information we track inside the interest, but really it is part of that information

**Comment by kialj876** [comment]
Looking really good, but the warning message box isn't coming up for me when I don't select anyone from the dropdown?

File: btr-web/btr-main-app/components/individual-person/IndividualConnection.vue [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+const activeIndividuals = computed(() => {
+  return props.significantIndividuals.filter((si) => {
+    return si.uuid !== model.value.uuid && si.ui.action !== FilingActionE.REMOVE
+  }).map((si) => {
+    return { name: si.name.fullName.toUpperCase(), uuid: si.uuid }
+  })
+})
+
+// selected inviduals that have shares acting jointly with the current SI
+const individualsWithSharesActingJointly = computed({
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/IndividualConnection.vue:114]: same here, store ?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-167-gsp7s372.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-167-gsp7s372.web.app

**Review by kialj876 — APPROVED** [review]
Nice work patrick!

One last thing if it is easy before you merge can you try getting the width of the combo box to match the input? Right now it is a bit wider
And also try to get the hover state to go across the full dropdown? Looks like there's some padding preventing that in the new one
Screenshot 2024-06-04 at 12 06 33 PM

**Review by hfekete — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-167-gsp7s372.web.app
